docs/proxy/google_serp_proxy/nodejs_examples.md

```diff
@@ -67,7 +71,11 @@ Get a list of shopping results from the Czech Republic for keyword `Apple iPhone
 
     Apify.main(async() => {
         const password = process.env.APIFY_PROXY_PASSWORD;
-        const proxyUrl = `http://groups-GOOGLE_SERP:${password}@localhost:8000`;
+        const proxyConfiguration = await Apify.createProxyConfiguration({
+            password,
+            groups: ['GOOGLE_SERP'],
+            countryCode: 'CZ',
```

Comment:
same here, `SERP` uses the TLD in URL to determine the country, unless I'm mistaken.